pipeline/transformer/transformer_miro/src/main/scala/uk/ac/wellcome/platform/transformer/miro/transformers/MiroContributors.scala

```diff
@@ -1,6 +1,6 @@
 package uk.ac.wellcome.platform.transformer.miro.transformers
 
-import uk.ac.wellcome.models.work.internal.{Agent, Contributor, Unminted}
+import uk.ac.wellcome.models.work.internal.{Agent, Contributor, IsState}
```

Comment:
Assuming `IdState`?

Reply:
Yep

Reply:
Working on another branch at the moment, expecting a fair few failures on this so probably should have marked as WIP